workers/network-manager/src/manager.rs

```diff
@@ -267,7 +267,9 @@ impl NetworkManagerExecutor {
                             log::info!("Listening on {}/p2p/{}\n", address, self.local_peer_id);
                         },
                         // This catchall may be enabled for fine-grained libp2p introspection
-                        _ => {  }
+                        e => {
+                            log::info!("{:?}", e);
+                        }
```

Comment:
seems this is likely not meant to be in this pr

Reply:
ah ack